borzalom/xcite
# Prevent nutzoid behaviour after choosing Dutch language

To test:
- Reset all settings
- Quit XCITE
- Restart
- Change language to dutch
- Click around and ensure working as expected
